Frida/Error.swift

```diff
@@ -14,4 +14,5 @@ public enum Error: Swift.Error {
     case notSupported(String)
     case protocolViolation(String)
     case transport(String)
+    case rpcError(String)
```

Comment:
Wonder if we can support the `stack` property somehow. (Fourth element of the `frida:rpc` tuple, which is optional.)

Reply:
Looking into it